src/Avalonia.Animation/IAnimator.cs

```diff
@@ -16,6 +16,6 @@ public interface IAnimator : IList<AnimatorKeyFrame>
         /// <summary>
         /// Applies the current KeyFrame group to the specified control.
         /// </summary>
-        IDisposable Apply(Animation animation, Animatable control, IObservable<bool> obsMatch, Action onComplete);
+        IDisposable Apply(Animation animation, Animatable control, IClock clock, IObservable<bool> obsMatch, Action onComplete);
```

Comment:
Just noticing this now, but `obsMatch` is a little obscure as a parameter name. Might be worth cleaning that up.